PRIVACY.md

```diff
@@ -0,0 +1,27 @@
+**Pontoon Add-on**:
+
+- **Does not** collect any personally identifiable data.
+- **Does not** communicate with its author.
+- **Does** download data from external servers, which are necessary to provide its functionality as described below.
+- **Does** store the download data locally as described below.
```

Comment:
I find this harder to read than splitting it into two blocks like this:

Pontoon Add-on does not:
-
-

Pontoon Add-on does:
-
-

In its current form, I read the 3rd and 4th line as "Does not" at first. But maybe that's just me.

Reply:
Let me split that in #150.